documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -1,35 +1,32 @@
 [[raspi-config]]
-== The `raspi-config` Tool
+== `raspi-config`
 
-`raspi-config` is the Raspberry Pi configuration tool, originally written by https://github.com/asb[Alex Bradbury]. To open the configuration tool, type the following on the command line:
+TIP: Raspberry Pi Desktop users can also access a graphical version of this application at **Preferences** > **Raspberry Pi Configuration**.
```

Comment:
(although there are some "advanced options" which are only in the CLI tool and not the GUI tool)

Reply:
I was trying to find the best way to bring this up. Probably an extra sentence here will do!